cmd/backup-manager/app/cmd/backup.go

```diff
@@ -42,10 +43,15 @@ func NewBackupCommand() *cobra.Command {
 
 	cmd.Flags().StringVar(&bo.Namespace, "namespace", "", "Backup CR's namespace")
 	cmd.Flags().StringVar(&bo.BackupName, "backupName", "", "Backup CRD object name")
+	cmd.Flags().StringVar(&bo.Host, "host", "", "Tidb cluster access address")
+	cmd.Flags().Int32Var(&bo.Port, "port", bkconstants.DefaultTidbPort, "Port number to use for connecting tidb cluster")
+	cmd.Flags().StringVar(&bo.Password, bkconstants.TidbPasswordKey, "", "Password to use when connecting to tidb cluster")
+	cmd.Flags().StringVar(&bo.User, "user", "", "User for login tidb cluster")
+	util.SetFlagsFromEnv(cmd.Flags(), bkconstants.BackupManagerEnvVarPrefix)
```

Comment:
Why add these options for`br` backup?  `br` not need these options.

Reply:
Yes. I use the same logic as mydumper.   backup-manager backup need these values to configure `tikv_gc_life_time`.  Do you have any suggestions? get these values in backup-manager image? I think mydumper could get these values in backup-manager image as well.